dashboard/longterm_extract.py

```diff
@@ -48,7 +48,7 @@ def download_truck_data_files(s3_client, target_bucket) -> list[str]:
 
 
 def extract_s3_files() -> list[str]:
-    """"""
+    """The main function."""
```

Comment:
You might be more specific here. Yes it is the main function, but what does it do/return?